DQM/SiPixelPhase1Common/src/HistogramManager.cc

```diff
@@ -546,6 +541,7 @@ void HistogramManager::executePerLumiHarvesting(DQMStore::IBooker& iBooker,
 }
 
 void HistogramManager::loadFromDQMStore(SummationSpecification& s, Table& t, DQMStore::IGetter& iGetter) {
+  //std::cout << "Running the loadFromDQMStore function" << std::endl;
```

Comment:
part of the debugging changes? Perhaps to remove?

Reply:
Removed!